### PR TITLE
Add keyboard support

### LINKED ISSUE #78: Keyboard support
This is probably most important for desktop usage but can also be useful on mobile if you have an external keyboard connected.

I would expect this to work out of the box:
ctrl+, ctrl-, ctrl0 to control zoom
arrow keys to pan when zoomed

### PR DESCRIPTION
**Pull request by evant**
Intial stab at this, I'm not sure all the code lives in the right place and looking for feedback on that.

Additional issues/questions:

- Is there a way to get the 'standard' system keyboard shortcuts for this?
- How to detect ctrl/meta + key press? I never saw them detected though I'm testing with the android emulator which does weird things with external keyboards.
- keyboard input enabled by default? note: view needs to be focused to start receiving events
- configure zoom & pan steps?

Fixes #78

### CODE REVIEW

File: sample/src/main/kotlin/me/saket/telephoto/sample/navigation.kt
Comment (evant) [inline sample/src/main/kotlin/me/saket/telephoto/sample/navigation.kt:26]: There's currently a bug where you can still keyboard focus the gallery screen when the media viewer is brought up, I'm not familiar enough with circuit to know what the right fix was so just changed it to render only the current page while testing. That does break the swipe away animation since it won't render the behind page until the animation completes

Comment (saket) [reply sample/src/main/kotlin/me/saket/telephoto/sample/navigation.kt:26]: > There's currently a bug where you can still keyboard focus the gallery screen when the media viewer is brought up

hmmmmmm I wonder if I can fix this by preventing touch/focus events from leaking behind the active screen.

**Review by saket — APPROVED** [review]
This is fantastic, thank you! 

Are you blocked on this by any chance? I'd like to complete https://github.com/saket/telephoto/issues/4 before I merge this, but I can try finding a stop-gap if you need this in your project.

**Comment by evant** [comment]
> Are you blocked on this by any chance?

Nope! We ended up adding separate zoom/pan support on top for keyboard support for now. It does mean you could technically do both independently but it seems to work ok enough.

File: sample/src/main/kotlin/me/saket/telephoto/sample/navigation.kt
Comment (evant) [reply sample/src/main/kotlin/me/saket/telephoto/sample/navigation.kt:26]: On my side project where I'm using a custom navigation solution I solved this by not rendering the behind page unless you were in the middle of the transition.

**Comment by saket** [comment]
Took me some time, but I'm finally working on this. I'm going ahead and merging this as-is because your code looks great. I'll make a few adjustments and then land the changes on `trunk` soon. Thanks Eva!

File: zoomable/src/commonMain/kotlin/me/saket/telephoto/zoomable/internal/keyboardActions.kt
Comment (saket) [inline zoomable/src/commonMain/kotlin/me/saket/telephoto/zoomable/internal/keyboardActions.kt:77]: I'm curious, what was the reason for supporting `Key.Zero`? I haven't been able to find any other app/OS that resets zoom when the `zero` key is pressed.

Comment (evant) [reply zoomable/src/commonMain/kotlin/me/saket/telephoto/zoomable/internal/keyboardActions.kt:77]: Because my browser does, didn't think too much about it and don't have strong opinions

**Pull request merged**
